src/main/kotlin/no/nav/syfo/client/OppgaveClient.kt

```diff
@@ -107,7 +108,7 @@ class OppgaveClient constructor(private val url: String, private val oidcClient:
             oppgavetype = "FDR",
             behandlingstype = behandlingstype,
             aktivDato = LocalDate.now(),
-            fristFerdigstillelse = LocalDate.now().plusDays(1),
+            fristFerdigstillelse = finnFristForFerdigstillingAvOppgave(LocalDate.now()),
```

Comment:
Her har du kun endret for fordelingsoppgave, du må endre for journalføringsoppgave også :)

Reply:
I fix